src/Collateral.sol

```diff
@@ -30,6 +29,8 @@ contract Collateral {
         uint256 thawEndTimestamp; // Block number at which thawing period ends
     }
 
+    uint256 private constant MAX_UINT256 = 2 ** 256 - 1;
```

Comment:
`type(uint256).max` is another option if you wanted to avoid defining this variable.

Reply:
I was following what was in the existing AllocationExchange contract, but I like this solution better.